contracts/coins/AvaxWalletSimple.sol

```diff
@@ -0,0 +1,312 @@
+ *
+ * The first signature is created on the operation hash (see Data Formats) and passed to sendMultiSig/sendMultiSigToken
+ * The signer is determined by verifyMultiSig().
+ *
+ * The second signature is created by the submitter of the transaction and determined by msg.signer.
+ *
+ * Data Formats
+ * ============
+ *
+ * The signature is created with ethereumjs-util.ecsign(operationHash).
+ * Like the eth_sign RPC call, it packs the values as a 65-byte array of [r, s, v].
+ * Unlike eth_sign, the message is not prefixed.
+ *
+ * The operationHash the result of keccak256(prefix, toAddress, value, data, expireTime).
+ * For ether transactions, `prefix` is "ETHER".
+ * For token transaction, `prefix` is "ERC20" and `data` is the tokenContractAddress.
+ *
+ *
+ */
+contract AvaxWalletSimple {
+  // Events
+  event Deposited(address from, uint value, bytes data);
+  event SafeModeActivated(address msgSender);
+  event Transacted(
+    address msgSender, // Address of the sender of the message initiating the transaction
+    address otherSigner, // Address of the signer (second signature) used to initiate the transaction
+    bytes32 operation, // Operation hash (see Data Formats)
+    address toAddress, // The address the transaction was sent to
+    uint value, // Amount of Wei sent to the address
+    bytes data // Data sent when invoking the transaction
+  );
+
+  // Public fields
+  address[] public signers; // The addresses that can co-sign transactions on the wallet
+  bool public safeMode = false; // When active, wallet may only send to signer addresses
+
+  // Internal fields
+  uint constant SEQUENCE_ID_WINDOW_SIZE = 10;
+  uint[10] recentSequenceIds;
+
+  /**
+   * Set up a simple multi-sig wallet by specifying the signers allowed to be used on this wallet.
+   * 2 signers will be required to send a transaction from this wallet.
+   * Note: The sender is NOT automatically added to the list of signers.
+   * Signers CANNOT be changed once they are set
+   *
+   * @param allowedSigners An array of signers on the wallet
+   */
+  function AvaxWalletSimple(address[] allowedSigners) public {
+    if (allowedSigners.length != 3) {
+      // Invalid number of signers
+      revert();
+    }
+    signers = allowedSigners;
+  }
+
+  /**
+   * Determine if an address is a signer on this wallet
+   * @param signer address to check
+   * returns boolean indicating whether address is signer or not
+   */
+  function isSigner(address signer) public view returns (bool) {
+    // Iterate through all signers on the wallet and
+    for (uint i = 0; i < signers.length; i++) {
+      if (signers[i] == signer) {
+        return true;
+      }
+    }
+    return false;
+  }
+
+  /**
+   * Modifier that will execute internal code block only if the sender is an authorized signer on this wallet
+   */
+  modifier onlySigner {
+    if (!isSigner(msg.sender)) {
```

Comment:
same. use `require` instead of `revert`

Reply:
`require` is not supported in Solidity `0.4.18`
We want to deviate the least possible from the original v2 contract implemented in [WalletSimple.sol](https://github.com/BitGo/eth-multisig-v2/blob/master/contracts/WalletSimple.sol), that way we can get a faster security audit.